elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -176,111 +225,127 @@
 /// ===========================================================================
 /// Component Specific SASS Vars
 /// ===========================================================================
-
-$pfe-expand_button--LineHeight:      1.5;
+$pfe-expand-button--LineHeight:      1.5;
 $pfe-expand_chevron--size:           0.313em;
 $pfe-expand_chevron--weight:         0.15em;
 $pfe-expand_chevron--placement:      ($pfe-expand_button--LineHeight / 2) - ($pfe-expand_chevron--size / 2);
 $pfe-expand_button--padding-factor:  0.75;
-$pfe-expand_button--border-dark:     #979797;
 $pfe-expand_button--padding: calc(#{pfe-var(container-spacer)} * #{$pfe-expand_button--padding-factor});
 
+@mixin pfe-accordion-header {
+  display: block;
+  //background: var(--pfe-accordion--main);
+  //color: var(--pfe-accordion--aux);
+
+  button {
+    -webkit-appearance: button;
+  }
+  > * {
+    margin: 0;
+  }
+}
+
 @mixin pfe-trigger-button($theme: light, $align: left) {
+    --pfe-accordion--main:               transparent;
+    --pfe-accordion--aux:                #{pfe-color(surface--lightest--text)};
+    --pfe-accordion--focus:              #{pfe-color(surface--lightest--link--focus)};
+    --pfe-accordion--border-left:        3px solid transparent;
+    --pfe-accordion--border-right:       1px solid transparent;
+    --pfe-accordion--border-top-bottom:  0px;
+    --pfe-accordion--border-focus-color: #{pfe-color(surface--accent)};;
```

Comment:
idk if accent is a desirable color but i wanted the focus state to be visible whether it was expanded or not